contracts/test/PresaleMock.sol

```diff
@@ -57,6 +57,10 @@ contract PresaleMock is IPresale {
         return collateralToken;
     }
 
+    function setCollateralToken(ERC20 _collateralToken) external {
```

Comment:
Why do we need this change in this PR?

Reply:
It's only a mock, so it was to make things easier and tests lighter, but yes, I can re-deploy everything with the new token.